README.md

```diff
@@ -13,11 +13,16 @@ initialBalance = 20  // balance to start with at launch
 levels = 30  // length of nameID (levels of skip graph)
 Mode = True // honest or malicious
 validationFees = 1 // reward received by validator
+Token = 20 //tokens if greater than 10 then passing the current contract ( logic behind transf.sol)
```

Comment:
all comments should have an empty space after `//`, i.e., 
`//tokens..` --> `// tokens...`
please check other places in your code as well.